src/main/java/projectbookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,15 +1,22 @@
 package projectbookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Getter;
 import lombok.Setter;
 
 @Getter
 @Setter
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
```

Comment:
also let's add `@NotBlank` to ensure that we do not treat `""` or `"   "` like it is a valid data